assets/js/plan_template.js

```diff
@@ -89,14 +89,15 @@ function tableToSlots() {
 async function postPlanTemplate() {
     document.getElementById("searchSpinner").classList.remove("visually-hidden");
     const location = document.getElementById('location').value.toString();
-    const locationFields = location.split(",");
+    const locationFields = location.split(", ");
```

Comment:
I see why we use ", " instead of "," as the separation symbol, because it takes care of the extra spaces in elements of `locationFields`.
Maybe we could use `locationFields = location.split(",").trim()`, because it can work with more cases?

Reply:
Great suggestion, but we cannot directly call trim after `split`. Added a `map` for trimming the fields